### ISSUE TITLE
`wasmparser` (indirect dep) about to break

### ISSUE THREAD

**Issue by teh-cmc** [rerun-io/rerun#2022]
`cargo` ain't happy about `wasmparser`:
```
warning: the following packages contain code that will be rejected by a future version of Rust: wasmparser v0.59.0, wasmparser v0.77.0
```

```
$ cargo tree --all --all-features -i wasmparser

error: There are multiple `wasmparser` packages in your project, and the specification `wasmparser` is ambiguous.
  wasmparser@0.59.0
  wasmparser@0.77.0
```

```
$ cargo tree --all-features -i wasmparser@0.59.0

wasmparser v0.59.0
└── wit-validator v0.2.1
    └── wasm-bindgen-cli-support v0.2.84
        └── cargo-run-wasm v0.3.2
            └── run_wasm v0.6.0-alpha.0 (/home/cmc/dev/rerun-io/rerun/run_wasm)
```

```
$ cargo tree --all-features -i wasmparser@0.77.0

wasmparser v0.77.0
└── walrus v0.19.0
    ├── wasm-bindgen-cli-support v0.2.84
    │   └── cargo-run-wasm v0.3.2
    │       └── run_wasm v0.6.0-alpha.0 (/home/cmc/dev/rerun-io/rerun/run_wasm)
    ├── wasm-bindgen-externref-xform v0.2.84
    │   └── wasm-bindgen-cli-support v0.2.84 (*)
    ├── wasm-bindgen-multi-value-xform v0.2.84
    │   └── wasm-bindgen-cli-support v0.2.84 (*)
    ├── wasm-bindgen-threads-xform v0.2.84
    │   └── wasm-bindgen-cli-support v0.2.84 (*)
    ├── wasm-bindgen-wasm-conventions v0.2.84
    │   ├── wasm-bindgen-cli-support v0.2.84 (*)
    │   ├── wasm-bindgen-threads-xform v0.2.84 (*)
    │   └── wasm-bindgen-wasm-interpreter v0.2.84
    │       └── wasm-bindgen-cli-support v0.2.84 (*)
    ├── wasm-bindgen-wasm-interpreter v0.2.84 (*)
    └── wit-walrus v0.6.0
        └── wasm-bindgen-cli-support v0.2.84 (*)
```

**Comment by emilk** [comment]
I guess we need to wait for a new release of cargo-run-wasm: https://crates.io/crates/cargo-run-wasm

**Comment by teh-cmc** [comment]
Fixed, we've been on `wasmparser` 0.70.1 for a while.
